test/common/config/config_provider_impl_test.cc

```diff
@@ -195,19 +195,11 @@ class DummyConfigProviderManager : public ConfigProviderManagerImplBase {
 
   // Envoy::Config::ConfigProviderManager
   ConfigProviderPtr
-  createStaticConfigProvider(const Protobuf::Message& config_proto,
+  createStaticConfigProvider(std::vector<std::unique_ptr<const Protobuf::Message>>&& configs,
```

Comment:
nit: can the type be replaced with `ProtobufTypes::ConstMessagePtrVector`?

Reply:
thanks for the good catch, done in https://github.com/envoyproxy/envoy/pull/34985/commits/f465f61c8aac890bd0883524682dbfb6c738cac7